### ISSUE TITLE
Add posibility to show combined clan seeding points

### ISSUE THREAD

**Issue by pavelhoral** [fantinodavide/Squad_Whitelister#28]
We are interested in seeing clan wide seeding points / percentages. This might come handy when we want to incentivize whole clans to seed with combined points of their individual members (i.e. if two players seed a lot, it is enough for a whitelist for a clan of ten players).

I don't think this needs to be automated like the individual seeding reward, but I would like to see a a sum of seeding time for all clan members somewhere in the UI. The simplest solution that comes to mind is to simply add "clan dropdown" on the "Seeding" page:

![image](https://github.com/user-attachments/assets/8a6a0aff-1676-4bcc-a9ba-4fe546f7b02a)

With this we can at least have an idea if at least some clan member are seeding or not. Of course it would be better to see the exact number of seeding percentage divided by the number of clan members...

**Comment by fantinodavide** [comment]
Hi @pavelhoral! I think this is an interesting suggestion, and I love it! I'm definitely going to start working on it soon!

**Comment by fantinodavide** [comment]
I'd be interested to see what other people think about the position you suggested. I was thinking in the Clan card (which will be renamed to Group at some point, and Groups renamed to Roles or something along the lines)

**Comment by pavelhoral** [comment]
I agree that seeing the clan percentage would make life much easier. This can be implemented as follows:

* add percentage to the _clan card_ that will be calculated as `(sum_of_members_seeding_points * 100) / (required_reward_player_time * number_of_clan_members)`
* adding proposed clan filter to the _Seeding_ page to allow inspecting seeding time of individual clan members

As I stated in the original post - I don't think this requires any automatic group membership resolution. Checking how clans are doing and acting on that information will be manual work.
